src/Util.js

```diff
@@ -27,6 +27,18 @@ class Util {
         };
     }
 
+    /**
+     * Returns the token alias.
+     *
+     * @return {Object} token alias protobuf
+     */
+    static tokenAlias() {
+        return {
+            type: 'DOMAIN',
+            value: 'token.io'
```

Comment:
We should hardcode this in constants.js

Reply:
It doesn't have to be a method, remove from Util

Reply:
what constants.js?